platform-friends-hybrid/styles/main.css

```diff
@@ -32,585 +46,610 @@
 .km-flat li.km-state-active .km-listview-label,
 .km-flat .km-listview-label input[type=radio]:checked,
 .km-flat .km-listview-label input[type=checkbox]:checked {
-    background: #3598db;
+  background: #4d4d4d;
+}
+
+.km-flat .km-loader {
+  background-color: rgba(0, 0, 0, .75);
+}
+
+.km-flat .km-loader::before {
+  background: none;
+}
+
+#friendsApp.km-flat,
+#friendsApp.km-flat .km-navbar,
+#friendsApp.km-flat .km-content,
+input[type='date'],
+input[type='datetime-local'],
+input[type='month'],
+input[type='time'],
+input[type='week'] {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Not sure if this is needed - the original Flat skin would probably use Roboto Light if available anyway (unless we want the normal one specifically):
https://github.com/telerik/kendo-ui-core/blob/master/styles/mobile/flat/base.less#L9